content/en/service_management/incident_management/notification.md

```diff
@@ -11,7 +11,14 @@ further_reading:
 
 ## Overview
 
-All stakeholder notifications for an incident are consolidated in the incident’s Notifications tab. You can manually create, save as draft, and send notifications directly from this page. Automated notifications sent by [Notification Rules][1] for the incident in question are also listed in this section.
+
+Effective incident response depends on notifying the right people at the right time. Datadog Incident Management provides two key ways to coordinate communication during an incident:
+
+- The **Notifications** tab centralizes stakeholder communications. From here, you can create and send manual updates, save drafts, and view all automated messages triggered by [Notification Rules][1].
+
+- The **Pages** tab helps you manage on-call escalations. From this tab, you can trigger new pages to alert responders based on your [Datadog On-Call][4] teams. This tab also shows a history of all pages sent, whether manually or through [Notification Rules][1], so you can track who has been paged and when.
```

Comment:
```suggestion
- The **Pages** tab helps you manage your on-call pages. From this tab, you can page [Datadog On-Call][4] teams to prompt them to join the incident response. This tab also shows a history of all pages sent, whether manually or through [Notification Rules][1], so that you can track which teams have been paged and when.
```